src/gql/graphql.ts

```diff
@@ -683,6 +684,7 @@ export type PreparedBuildTestsArgs = {
   first?: InputMaybe<Scalars['Int']['input']>;
   last?: InputMaybe<Scalars['Int']['input']>;
   orderBy?: InputMaybe<PreparedBuildTestsOrder>;
+  statuses?: InputMaybe<Array<TestStatus>>;
```

Comment:
I think this is a good example of graphql updating unrelated because a change was made on chromatic, not in the addon itself. I would have thought this would have updated in [this pr that queried for tests by statuses](https://github.com/chromaui/addon-visual-tests/pull/37/files#diff-6057cadf03bd6137c028bb3632be21876ce4b2773596ac70aff9a2ba2caad01e)